js/coinflex.js

```diff
@@ -0,0 +1,2384 @@
+        const clientOrderId = this.safeString (params, 'clientOrderId');
+        const maxCOI = '9223372036854775807';
+        if ((clientOrderId !== undefined) && Precise.stringGt (clientOrderId, maxCOI)) {
+            throw new InvalidOrder (this.id + ' createOrder() param clientOrderId should not exceed ' + maxCOI);
+        }
+        const orderType = this.convertOrderType (type);
+        // creating stop orders using type argument will mess up the unification logic (beacuse of missing market/limit). So, we have to use unified approach for sending stop orders
+        if (orderType === 'STOP') {
+            throw new ArgumentsRequired (this.id + ' createOrder() : to create a stop order, you need to specify the "stopPrice" param');
+        }
+        await this.loadMarkets ();
+        const order = {
+            'marketCode': market['id'],
+            'side': this.convertOrderSide (side),
+            'orderType': this.convertOrderType (type),
+            'quantity': this.amountToPrecision (market['symbol'], amount),
+        };
+        const stopPrice = this.safeNumber (params, 'stopPrice');
+        const isStopOrder = stopPrice !== undefined;
+        if (isStopOrder) {
+            order['stopPrice'] = this.priceToPrecision (market['symbol'], stopPrice);
+            params = this.omit (params, 'stopPrice');
+        }
+        if (price !== undefined && type === 'limit') {
+            // stop orders have separate field for limit price
+            if (isStopOrder) {
+                let limitPrice = this.safeNumber (params, 'limitPrice');
+                if (limitPrice !== undefined) {
+                    params = this.omit (params, 'limitPrice');
+                } else {
+                    limitPrice = price;
+                }
+                order['limitPrice'] = this.priceToPrecision (market['symbol'], limitPrice);
+            } else {
+                order['price'] = this.priceToPrecision (market['symbol'], price);
+            }
+        }
+        const request = {
+            'responseType': 'FULL', // FULL, ACK
+            'orders': [ order ],
+        };
+        return [ request, params ];
+    }
+
+    checkOrderResponseForException (response, firstResult) {
+        const success = this.safeValue (firstResult, 'success');
+        if (success === 'false') {
+            const message = this.safeString (firstResult, 'message');
+            const code = this.safeString (firstResult, 'code');
+            const body = this.id + ' ' + this.json (response);
+            this.throwExactlyMatchedException (this.exceptions['exact'], code, body);
+            this.throwBroadlyMatchedException (this.exceptions['broad'], message, body);
+            throw new ExchangeError (body);
+        }
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        const market = this.market (symbol);
+        this.checkOrderTypeAndPrice (market, type, side, amount, price);
```

Comment:
It is a bit dangerous to add features that are not available yet, make sure the other PR is merged before this one. 

IMO you could merge the other branch here and although you would see those changes in Exchange.js/py/php here they would disappear after the other PR being merged and even if by accident this PR was merged first than the one introducing the feature, nothing would break

Reply:
ah, right! done